docs/docs/quickstart-inference.md

```diff
@@ -0,0 +1,262 @@
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+# Learn Weave with W&B Inference
+
+This guide shows you how to use W&B Weave with [W&B Inference](https://docs.wandb.ai/guides/inference/). Using W&B Inference, you can build and trace LLM applications using live open-source models without setting up your own infrastructure or managing API keys from multiple providers. Just obtain your W&B API key and use it to interact with [all models hosted by W&B Inference](https://docs.wandb.ai/guides/inference/models/).
+
+## What you'll learn
+
+In this guide, you'll:
+- Set up Weave and W&B Inference
+- Build a simple LLM application with automatic tracing
+- Compare multiple models
+- Evaluate model performance on a dataset
+- View your results in the Weave UI
+
+## Prerequisites
+
+Before you begin, you need a [W&B account](https://app.wandb.ai/login?signup=true) and an API key from from [https://wandb.ai/authorize](https://wandb.ai/authorize).
+
+Then, in a Python environment running version 3.8 or later, install the required libraries: 
+
+```bash
+pip install weave openai
+```
+
+The `openai` library is installed because you use the standard `openai` client to interact with W&B Inference, regardless of which hosted model you're actually calling. This allows you to swap between supported models by only changing the slug, and make use of any existing code you have that was written to use the OpenAI API. 
+
+## Step 1: Trace your first LLM call
+
+Start with a simple example that uses Llama 3.1-8B through W&B Inference. 
```

Comment:
```suggestion
Start with a basic example that uses Llama 3.1-8B through W&B Inference. 
```
This is a nit, but I try to avoid the word "simple" because it's a subjective opinion and the example might not be simple to everyone. IMO, basic works better.